otterdog/eclipse-velocitas.jsonnet

```diff
@@ -317,6 +317,25 @@ orgs.newOrg('eclipse-velocitas') {
         },
       ],
     },
+    orgs.newRepo('vehicle-app-kotlin-sdk') {
+      allow_update_branch: false,
+      description: "Vehicle App sdk for Kotlin",
+      has_wiki: false,
+      is_template: false,
+      web_commit_signoff_required: false,
+      workflows+: {
+        default_workflow_permissions: "write",
```

Comment:
I would strongly suggest to use read permissions by default.

Reply:
Migrating existing workflows to use read settings by default might be tedious, but when creating new repos it is quite straightforward. At some point we want to use read permissions by default, but its up to you how to do it right now.

Reply:
lets keep it synced over all repos for now and change all together later

Reply:
as you prefer